src/main.ts

```diff
@@ -0,0 +1,18 @@
+import * as globby from 'globby';
+
+/**
+ * Finds all files in the given path that match the given extension.
+ * Looks through directories and sub directories. Default extension is '.test.js'.
+ * @param path the path to search in
+ * @param extensions a string array of extensions to look for
+ * @returns a promise with an array of paths to found files
+ */
+
+export async function findFiles(path: string, extensions: string[] = ['.test.js']): Promise<string[]> {
```

Comment:
Please call `npm run lint` it will format the code using **prettier**.
https://prettier.io

Reply:
Done, thanks